rust/src/rfb/rfb.rs

```diff
@@ -226,7 +243,11 @@ impl RFBState {
                             consumed += current.len() - rem.len();
                             current = rem;
 
-                            self.state = parser::RFBGlobalState::TCSecurityResult;
+                            if let Some(frame) = &pdu {
+                                frame.set_len(flow, consumed as i64);
+                            }
+
+                        self.state = parser::RFBGlobalState::TCSecurityResult;
```

Comment:
nit: indent

Reply:
Same issue as I mentioned yesterday. The commit appears differently in my editor where this line has a different indent level.

Reply:
I think this was happening due to formatting in the wrong commit. I will fix it in the next PR